ibis-server/tests/routers/v2/connector/test_trino.py

```diff
@@ -371,3 +371,35 @@ def to_connection_info(trino: TrinoContainer):
 def to_connection_url(trino: TrinoContainer):
     info = to_connection_info(trino)
     return f"trino://{info['user']}@{info['host']}:{info['port']}/{info['catalog']}/{info['schema']}"
+
+
+def test_metadata_list_tables(self, trino: TrinoContainer):
+    connection_info = self.to_connection_info(trino)
+    response = client.post(
+        url=f"{self.base_url}/metadata/tables",
+        json={
+            "connectionInfo": connection_info,
+        },
+    )
+    assert response.status_code == 200
+
+    result = response.json()[0]
+    assert result["name"] is not None
+    assert result["columns"] is not None
+    assert result["primaryKey"] is not None
+    assert result["description"] is not None
+    assert result["properties"] is not None
+
+
+def test_metadata_list_constraints(self, clickhouse: TrinoContainer):
```

Comment:
You can use `just test trino` to test these codes.
Remember to rename the variable name when you copy it.

Reply:
Hey, @grieve54706  Can you look into test on my behalf. Cause I am not sure what is going wrong?

Reply:
@grieve54706  No need to worry all the taste-cases is being passed now.